flake.nix

```diff
@@ -143,6 +143,7 @@
               stylish-haskell.enable = true;
               typos.enable = true;
               yamllint.enable = true;
+              editorconfig-checker.enable = true;
```

Comment:
Consider sorting the entries by placing `editorconfig-checker` between `deadnix` and `hlint`.